libraries/oneview_helper.rb

```diff
@@ -168,7 +168,11 @@ def self.convert_keys(info, conversion_method)
       return nil unless info
       support = {}
       info.each do |k, v|
-        con = convert_keys(v, conversion_method) if v && v.class == Hash
+        if v && v.class == Hash
```

Comment:
just an observation: It is not necessary verify the `if v` because if `v` is a `Hash`, it is not `nil`.